tools/fast/fastesting/basic.go

```diff
@@ -0,0 +1,132 @@
+package fastesting
+
+import (
+	"context"
+	"io/ioutil"
+	"math/big"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/ipfs/go-ipfs-files"
+	"github.com/stretchr/testify/require"
+
+	"github.com/filecoin-project/go-filecoin/testhelpers"
+	"github.com/filecoin-project/go-filecoin/tools/fast"
+	"github.com/filecoin-project/go-filecoin/tools/fast/series"
+	localplugin "github.com/filecoin-project/go-filecoin/tools/iptb-plugins/filecoin/local"
+)
+
+type TestEnvironment struct {
```

Comment:
Please add documentation comments to these types and methods, as it looks like they're intended to be widely used by test writers.

In fact, why did the linter not catch this?

Reply:
I'm a little surprised that this is passing CI now. The linter should of said something about these missing.